src/pages/guides/using/creating_actions.md

```diff
@@ -4,7 +4,7 @@ For your code to execute as an *action* on Adobe I/O Runtime, your code has to c
 - You either call your function *main* or you export the entry point as *main*. This is the function that will be executed when it is invoked.
 - Your function accepts valid JSON objects as input and produces valid JSON objects as output, if needed
 
-You have to configure two tools on your machine to create and invoke actions from your machine: **wsk CLI** and **wskdeploy CLI.** Refer to the [wsk CLI](../tools/wsk_install.md) page for how to install and configure it. 
+You have to configure two tools on your machine to create and invoke actions from your machine: **aio CLI** and **wskdeploy CLI.** Refer to the [aio CLI](../tools/cli_install.md) page for how to install and configure it. 
```

Comment:
Maybe we can move `wskdeploy CLI` to the section where it's used?
1. Here we mention it, but don't provide a link to the docs how to install it
2. it's not used for single action deployment, so it's not needed for all use cases (but we say `You have to` here).

It's another discussion whether we still need to reference this tool or provide steps for useing `aio` CLI for deploying complex actions.

Reply:
`wskdeploy` is used in the same file, I'm not sure what is the case for it, so decided to don't touch it.

Reply:
I mean, it's used in one of the sections ("Deploying from ZIP archive" or so"). I thought maybe we can reference it there. But not critical.

Reply:
I changed it.